ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -452,6 +459,35 @@ objects:
         sshKey: |
           ${SSH_PUB_KEY}
         EOF
+        elif [[ "${CLUSTER_TYPE}" == "azure4" ]]; then
+            cat > /tmp/artifacts/installer/install-config.yaml << EOF
+        apiVersion: v1beta4
+        baseDomain: ci.azure.devcluster.openshift.com
+        metadata:
+          name: ${CLUSTER_NAME}
+        controlPlane:
+          name: master
+          replicas: 3
+        compute:
+        - name: worker
+          replicas: 3
```

Comment:
I don't see a need to set `controlPlane` or `compute` unless we're overriding defaults.  Same for `networking` below (I've filed #3914 to drop unnecessary `networking` entries from master).